treeherder/webapp/api/pagination.py

```diff
@@ -1,34 +1,6 @@
-from django.core.paginator import Paginator
-from django.utils.functional import cached_property
 from rest_framework import pagination
-from rest_framework.response import Response
 
 
 class IdPagination(pagination.CursorPagination):
     ordering = ('-id')
     page_size = 100
-
-
-# Django's Paginator class uses queryset.count() which
-# performs a full table scan
-class CustomPaginator(Paginator):
```

Comment:
I feel like this would be useful to leave in for future reference, but it's not being used anymore....